include/openssl/ec.h

```diff
@@ -144,6 +144,11 @@ OSSL_DEPRECATEDIN_3_0 const EC_METHOD *EC_GFp_nistp224_method(void);
  */
 OSSL_DEPRECATEDIN_3_0 const EC_METHOD *EC_GFp_nistp256_method(void);
 
+/** Returns 64-bit optimized methods for nistp384
+ *  \return  EC_METHOD object
+ */
+OSSL_DEPRECATEDIN_3_0 const EC_METHOD *EC_GFp_nistp384_method(void);
+
```

Comment:
Just avoid adding the public function and create an internal non-deprecated function. IMO there is no need to have this function public.

Reply:
Done! Thanks